Calculator/Calculator/ViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by yudonlee on 2022/05/27.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
최종 class 앞에는 `final`을 붙여준다면 더이상 오버라이드가 안된다는 사실을 알려주어 성능 향상이 돼요!
```suggestion
class ViewController: UIViewController {
```
```suggestion
final class ViewController: UIViewController {
```

Reply:
WWDC16 Understanding Swift Performance